<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>kmm</class>
 <widget class="QMainWindow" name="kmm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kindle Manga Maker 0.1</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/manga-icon.png</normaloff>:/icons/manga-icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="kmm_main_tab_widget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>674</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Project</string>
    </property>
    <addaction name="actNew"/>
    <addaction name="actOpen"/>
    <addaction name="actSave"/>
    <addaction name="actSaveAs"/>
    <addaction name="actClose"/>
    <addaction name="separator"/>
    <addaction name="actQuit"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>Export</string>
    </property>
    <addaction name="actionExport_to_cbr"/>
    <addaction name="actionExport_to_cbz"/>
    <addaction name="actionExport_to_epub"/>
    <addaction name="actionCreate_mobi"/>
    <addaction name="separator"/>
    <addaction name="actPreferences"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionTopics"/>
    <addaction name="separator"/>
    <addaction name="actAboutQt"/>
    <addaction name="actAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuRun"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="kmm_qstatusbar"/>
  <action name="actNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>Create a new project.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open a project file.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save the project in a file.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actSaveAs">
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="toolTip">
    <string>Save the project in a new file.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Exit the Program</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCreate_mobi">
   <property name="text">
    <string>Export to .mobi</string>
   </property>
  </action>
  <action name="actPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
   <property name="toolTip">
    <string>Set several preferences</string>
   </property>
  </action>
  <action name="actionTopics">
   <property name="text">
    <string>Help Topics</string>
   </property>
  </action>
  <action name="actAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actAbout">
   <property name="text">
    <string>About Kindle Manga Maker</string>
   </property>
  </action>
  <action name="actClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="toolTip">
    <string>Close the current tab.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionExport_to_epub">
   <property name="text">
    <string>Export to .epub</string>
   </property>
  </action>
  <action name="actionExport_to_cbr">
   <property name="text">
    <string>Export to .cbr</string>
   </property>
  </action>
  <action name="actionExport_to_cbz">
   <property name="text">
    <string>Export to .cbz</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>kmm_main_tab_widget</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
